### DISCUSSION TITLE
DIY zigbee bridge using WROOM32 and E72

### DISCUSSION THREAD

**Discussion by jf-wm** [arendst/Tasmota#17042] [Q&A]
Hi, I have an ESP32 module (wroom-32) and a CC2652p module (E72 2G4M20S1E) lying around. I would like to use both as a DIY zigbee bridge to work with the zbbrdpro binary. Unfortunately I can't find a schematic to understand the wiring needed between the two modules. Has anyone tried this before or have a source for a schematic so I can at least try?

**Comment by Jason2866** [comment]
You have to modify the zigbee firmware. See https://ptvo.info/zigbee-configurable-firmware-features/e72-2g4m20s1e-with-cc2652p/

If you follow the disscussion https://github.com/arendst/Tasmota/discussions/14419
you will probably have all infos you need.

**Comment by s-hadinger** [comment]
I will get back shortly with the pinout if you want to mimic the zbbridge pro.

But indeed you need a custom cc2652p firmware. Not sure if you can find them precompiled

**Comment by Jason2866** [comment]
[This](https://ptvo.info/download/ptvo-firmware-latest.zip) includes a working firmware and a tool to make the needed changes to work with the E72 module
